README.md

```diff
@@ -1,12 +1,12 @@
 # ![Shaka Player](docs/shaka-player-logo.png)
 
-Shaka Player is an open-source JavaScript library for adaptive media.  It plays
+Shaka Player is an open-source JavaScript library for adaptive media.  It plays
```

Comment:
Please revert all whitespace changes.  These obscure the real corrections and changes.